html/birds.html

```diff
@@ -3,9 +3,57 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Birds</title>
+    <link rel="stylesheet" href="../styles/Fonts/LYLAS_WOFF/LYLASFonts.css">
+    <link rel="stylesheet" href="../styles/Fonts/KoHo_WOFF/KohoFonts.css">
+    <link rel="stylesheet" href="../styles/general.css">
+    <link rel="stylesheet" href="../styles/global.css">
+    <link rel="stylesheet" href="../styles/variables.css">
+    <link rel="stylesheet" href="../styles/birds.css">
 </head>
+
 <body>
-    
+    <div class="container__section1">
+        <section class="section1">
+            <div class="section1__text">
+                <p class="section1__text--changeContent">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. 
+                    Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure 
+                    dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non 
+                    proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                </p>
+            </div>
+            <div class="section1__title">
+                <span class="section1__title--changeContent1">LOREM</span><br><span class="section1__title--changeContent2">IPSUM</span>
```

Comment:
instead of the changeContent2 you can add a "-" between the last letter and number for some semantic readability. 

section-1__title--changeContent-2

Reply:
"--" means it's a modifier, like "--active" or "--shadow-large". So i think in this case when it's identifies a span it might be "section-1__title-1" and "section-1__title-2" only.